common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4673,6 +4673,41 @@
     },
     "sqlState" : "0A000"
   },
+  "UNSUPPORTED_SHOW_CREATE_TABLE" : {
```

Comment:
Can we reuse `FORBIDDEN_OPERATION` ?

Reply:
@panbingkun It looks good, but I think that additional information needs to be told to the user for `WITH_UNSUPPORTED_FEATURE` and `WITH_UNSUPPORTED_SERDE_CONFIGURATION` error cases, should we use subclasses with more message under `FORBIDDEN_OPERATION`? and it will also bring some changes to the previous `FORBIDDEN_OPERATION` error.

Reply:
> Can we reuse `FORBIDDEN_OPERATION` ?

Perhaps we can add a `FORBIDDEN_OPERATION_WITH_REASON` as well? WDYT @panbingkun @wayneguow ?